src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -304,6 +303,9 @@ public static function collectDeprecations($outputFile)
      */
     public static function getPhpUnitErrorHandler()
     {
+        if (!isset(self::$isAtLeastPhpUnit83)) {
```

Comment:
this fixes the remaining error in travis `call_user_func() expects parameter 1 to be a valid callback, class 'PHPUnit\Util\ErrorHandler` because the method `register` were not called. Which is the case in our test suite when
`SYMFONY_DEPRECATIONS_SERIALIZE` it's defined its call `DeprecationErrorHandler::collectDeprecations`
and when `SYMFONY_DEPRECATIONS_HELPER` is `disabled` it's don't call `DeprecationErrorHandler::register`

Reply:
fix in #33087 to make this test green